packages/bpk-stylesheets/index.scss

```diff
@@ -18,6 +18,7 @@
 
 @import '~normalize.css';
 @import '~bpk-mixins/index';
+@import './_fonts.scss'; /* stylelint-disable-line scss/at-import-no-partial-leading-underscore */
```

Comment:
This can just be `@import 'fonts'` SASS is smart enough to figure this out

Reply:
Well that is clever! Learnt something new today